templates/admin/machine/index.html.ep

```diff
@@ -1,18 +1,26 @@
 % layout 'admin';
 % title 'Machines';
 
+% content_for 'ready_function' => begin
+$('#machines').DataTable( {
+"paging" : false,
+"lengthChange": false,
+"order": [],
+} );
+% end
+
```

Comment:
Has you actually tried the combination of DataTable and admintable? I think it does not work (at least out of the box) because the table is actually populated by javascript, which drives DataTable filtering and ordering crazy.